hypothesis-python/src/hypothesis/internal/conjecture/junkdrawer.py

```diff
@@ -152,8 +153,8 @@ def binary_search(lo, hi, f):
 
 
 def uniform(random, n):
-    """Returns an hbytes of length n, distributed uniformly at random."""
-    return int_to_bytes(random.getrandbits(n * 8), n)
+    """Returns a bytestring of length n, distributed uniformly at random."""
+    return random.getrandbits(n * 8).to_bytes(n, "big")
```

Comment:
`secrets.token_bytes()` is waiting just out of reach…

Reply:
Alas, not for us - we need to be able to seed the PRNG for reproducibility!

Reply:
TBH we're probably just going to delete this code soonish anyway.